390/Olivia_Handoko.Rmd

```diff
@@ -0,0 +1,320 @@
+---
+title: "390/Olivia_Handoko"
+output: html_document
+---
+ stem dying/broken below to dbh connection?
+- environment?
+-quadrant placement?
+
+same tree specie in different cooradinates, how does their dbh differ/if they are alive or if they are dead/other flaw  
+- maybe try a couple of different tree species but not all of them 
+  - based one species with most stem dead/broken below with those with the least stem dead/broken below in the latest data 
+See in each scbi_stem data to see which trees are the least and most for 
+- maybe pinpoint a place in the scatterplot or linegraph where the changes are if there is thatmightve affected in the timeline the die off of these trees 
+  - consider stem dead and broken below
+
+#notes in code to help choose which trees I guess 
+```{r}
+#libaries
+library(dplyr)
+library(tidyverse)
+library(tsibble)
+library(lubridate)
+library(fpp3)
+library(fpp2)
+library(ggfortify)
+library(lubridate)
+```
+
+```{r}
+#filter out from all scbi_stem data for DF status other than "alive" just to see which trees had the most stem dead and broken below in scbi_stem3 (year 2018)
+##
+
```

Comment:
Hey Olivia, here is a code review! 

A heads up that you didn't include the code that loads the csv, so unfortunately people wouldn't be able to reproduce your results. Could you include this code, being mindful of the file path as I say here https://github.com/390-2020-09-Fall/SCBI-ForestGEO-Data/pull/2#discussion_r491179003